custom/icds_reports/utils/aggregation_helpers/distributed/child_health_monthly.py

```diff
@@ -369,6 +373,10 @@ def _state_aggregation_query(self, state_id):
               AND agg_migration.month = %(start_date)s
               AND child_health.state_id = agg_migration.state_id
               AND child_health.supervisor_id = agg_migration.supervisor_id
+            LEFT OUTER JOIN "{agg_as_table}" agg_as ON child_health.mother_id = agg_as.person_case_id
```

Comment:
nit: agg_as is not an intuitive name. this might be clearer if you spelled it out a bit more, maybe `agg_availing`

Reply:
Done!